test/NuGet.Core.Tests/NuGet.ProjectModel.Test/LockFileFormatTests.cs

```diff
@@ -2447,6 +2447,126 @@ public void LockFileFormat_WritesCentralTransitiveDependencyGroups()
             Assert.Equal(expected.ToString(), output.ToString());
         }
 
+        [Fact]
+        public void LockFileFormat_WhenPackageSpecIsAvailableAppliesTargetAliasToLockFileTarget()
+        {
+            // Arrange
+            var lockFileContent = @"{
```

Comment:
If you used [raw string literals](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/raw-string), you could avoid needing double `""` every time the json needs a single `"`.

Reply:
I honestly forgot about them, I'll keep it in mind for future work since it's way simpler.